`get_iterations` sometimes returns an _unsorted_ list of iterations

This can lead to bad things like tricking us into thinking the time-steps are negative.

Leads to this error:

```julia
julia> include("quick_calibrate.jl")
[ Info: Precompiling OceanTurbulenceParameterEstimation [eca81dc5-87a6-4430-aec8-c76695404a43]
[ Info: Oceananigans will use 8 threads
┌ Warning: Package CubedSphere does not have CairoMakie in its dependencies:
│ - If you have CubedSphere checked out for development and have
│   added CairoMakie as a dependency but haven't updated your primary
│   environment's manifest file, try `Pkg.resolve()`.
│ - Otherwise you may need to report an issue with CubedSphere
└ Loading CairoMakie into CubedSphere from project dependency, future warnings for CubedSphere are suppressed.
[ Info: Oceananigans will use 8 threads
┌ Warning: HydrostaticFreeSurfaceModel is experimental. Use with caution!
└ @ Oceananigans.Models.HydrostaticFreeSurfaceModels ~/.julia/packages/Oceananigans/9o1xJ/src/Models/HydrostaticFreeSurfaceModels/hydrostatic_free_surface_model.jl:106
ERROR: LoadError: AssertionError: Simulation time steps are not uniformly spaced.
```

--- a/src/ModelsAndData/file_wrangling.jl
+++ b/src/ModelsAndData/file_wrangling.jl
@@ -1,6 +1,7 @@
 function get_iterations(datapath)
     file = jldopen(datapath, "r")
     iters = parse.(Int, keys(file["timeseries/t"]))
+    sort!(iters) # We always want iterations to be sorted from small to large
     close(file)
     return iters
 end

